pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
Got it! 

I don't remember why it was added. I think @dakrone might know.

So may be removing it instead of trying to fix it is better then?
Actually, with the multi module effort and what I started about building assembly modules, this won't work as `conf`, `lib` ... won't be part anymore of `core` project.

I think I would simply remove it from core for now and potentially add it again after the refactoring but in another module. We can have a module for that if there is a need IMO.